src/main/java/org/mifos/connector/mpesa/utility/ChannelRequestToMpesaMapper.java

```diff
@@ -0,0 +1,25 @@
+package org.mifos.connector.mpesa.utility;
+
+import org.mifos.connector.common.channel.dto.TransactionChannelRequestDTO;
+import org.mifos.connector.mpesa.dto.BuyGoodsPaymentRequestDTO;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ChannelRequestToMpesaMapper implements
+        EntityMapper<TransactionChannelRequestDTO, BuyGoodsPaymentRequestDTO> {
```

Comment:
nit: Entity and domain words are loosely used over here, I don't think these qualify as Entity. They're fundamentally different ideas. 

Typically, the domain object defines the business object and it's properties and methods. It's used to manipulate and move the data within the processing system. The Entity object exists to take those domain properties and map them to a persistent storage object, such as a database table.

Reply:
suggestion over the class name? what I should use? Mapper ?